getting_started/publish_app.md

```diff
@@ -0,0 +1,108 @@
+# Publishing your First Project Firefly Application
+In this tutorial, we'll guide you through the following steps to give you an introduction on how to publish your first Firefly Application. 
+after you deployed your application, you need to go through below 3 steps to publish your headful app
+1. Submit for approval through [Adobe Developer Console](https://console.adobe.io/)
+2. Review and approve the app in [My Exchange](https://exchange.adobe.com/my-exchange.html)
+3. After approval, your app will be available at [Adobe Experience Cloud](https://experience.adobe.com/#/@adobeio/home)
+
+## 1: Project approvals (Console Part)
+
+Once you have completed development on your project, and deploy your app in the Production workspace,your application is ready for approval. 
+
+There are three different approval processes within Adobe Developer Console, depending on the type of application that you have built:
+
+1. **Project Firefly applications:** Projects built using the Project Firefly template are built by an organization for use within that organization. Therefore, Firefly applications require approvals by the enterprise organization administrator only. 
+    
+    Please follow the approval process for [Project Firefly applications](#project-firefly-applications) outlined in this document.
+
+2. **Applications for publication:** Building an application for distribution to general users on the Adobe Exchange requires approval from the Adobe Review team before it can be published.
+
+    Please refer to the [applications for publication](#applications-for-publication) approval process outlined in this document.
+
+3. **XD Plugin distribution:** Currently only available as personal projects, XD plugins must be submitted for review and approval before they can be published to the Adobe XD Plugin Marketplace. 
```

Comment:
This is not relevant for Project Firefly. Could you mention it and remove `For detailed instructions on how to submit an XD plugin for approval, follow the steps provided in the [XD plugin distribution guide](plugin-distribution.md).`?